R/carehomes.R

```diff
@@ -77,6 +82,7 @@ carehomes_parameters <- function(start_date, region,
                                  react_sensitivity = 0.99,
                                  prop_noncovid_sympt = 0.01,
                                  rel_susceptibility = 1,
+                                 waning_rate = 0,
```

Comment:
Just to confirm: a zero rate collapses to the current model?

Reply:
Yes, zero rate means people will stay in R and never move back to S.

Reply:
yes